treeherder/perf/migrations/0018_remove_old_frameworks.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.2.4 on 2019-09-17 12:23
```

Comment:
We have to also delete the associated fixture references from [performance_framework.json](https://github.com/mozilla/treeherder/blob/master/treeherder/perf/fixtures/performance_framework.json#L35), otherwise `performance_framework` will be repopulated with these frameworks.

Reply:
Ok, thanks.